distribution/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -879,76 +878,62 @@ The response is returned as a data map that contains an internal map with the fo
- 					"title": "Get resource",
-					"description": "Gets the Metacard resource",
- 					"url": "http://${ddf-branding-lowercase}.codice.org:8181/services/catalog/sources/${ddf-branding-lowercase}.distribution/85227c45d9c34fe1ad3e725a72d6b44a?transform=resource"
- 				},
- 				{
- 					"id": "catalog.data.metacard.html",
-					"title": "Get html",
-					"description": "Gets the Metacard html",
-					"url": "http://${ddf-branding-lowercase}.codice.org:8181/services/catalog/sources/${ddf-branding-lowercase}.distribution/85227c45d9c34fe1ad3e725a72d6b44a?transform=html"
- 				} ]
- 		},
- 		...other metacards...
- 	],
- 	"status": [
- 	{
- 		"elapsed": 539,
-		"hits": 10,
-		"id": "${ddf-branding}-OS",
-		"results": 10,
-		"state": "SUCCEEDED"
- 	},
- 	{
- 		"elapsed": 11,
-		"hits": 10,
-		"id": "${ddf-branding-lowercase}.distribution",
-		"results": 10,
-		"state": "SUCCEEDED"
- 	}],
-	"types": {
- 		"${ddf-branding-lowercase}.metacard" : {
- 			"resource-uri" : "STRING",
-			"location" : "GEOMETRY",
-			"expiration" : "DATE",
-			"metadata-target-namespace" : "STRING",
-			"metadata-content-type" : "STRING",
-			"effective" : "DATE",
-			"modified" : "DATE",
-			"id" : "STRING",
-			"title" : "STRING",
-			"thumbnail" : "BINARY",
-			"created" : "DATE",
-			"metadata-content-type-version" : "STRING",
-			"resource-size" : "STRING",
-			"metadata" : "XML"
- 		},
- 		...other metacard types...
- 	}
- 	}
- }
+{
+    "data": {
+        "hits": 1,
+        "metacard-types": {
+            "ddf.metacard": {...}
+        },
+        "id": "857f76c0-da8c-72f5-09cd-536b69e427af",
+        "results": [
+           {
+                "metacard": {
+                    "cached": "2016-06-09T22:46:57.258+0000",
+                    "geometry": {
+                        "coordinates": [
+                            -97.03125,
+                            35.173808
+                        ],
+                        "type": "Point"
+                    },
+                    "type": "Feature",
+                    "actions": [ ... ],
+                    "properties": {
+                        "thumbnail": "..."
+                        "metadata": "<?xml version=\"1.0\" encoding=\"UTF-8\"?><metadata> ... </metadata>",
+                        "resource-size": "92314",
+                        "created": "2016-06-09T22:46:37.707+0000",
+                        "metacard-tags": [
+                            "resource"
+                        ],
+                        "resource-uri": "content:1a83781dd56a4050afc420016d078e22",
+                        "checksum-algorithm": "Adler32",
+                        "metadata-content-type": "image/jpeg",
+                        "metacard-type": "ddf.metacard",
+                        "title": "fixTheBugs_geotagged.jpg",
+                        "resource-download-url": "https://localhost:8993/services/catalog/sources/ddf.distribution/1a83781dd56a4050afc420016d078e22?transform=resource",
```

Comment:
Probably not a good idea. This is an example pulled right from the CometD endpoint. We want integrators to take it literally. Do we want a property substitution in the URL if you'd normally never see that?